app/presenters/service_manual_guide_presenter.rb

```diff
@@ -13,8 +13,12 @@ def initialize(content_item)
   end
 
   def content_owner
-    content_owner_data = content_item["details"]["content_owner"]
-    ContentOwner.new(content_owner_data["title"], content_owner_data["href"])
+    content_owner_data = Array(content_item["links"].try(:[], "content_owners")).first
```

Comment:
I'm not sure `.try(:[], key)` is a good pattern here. I think this might read better:

```ruby
if content_item["links"] && content_item["links"]["content_owners"].present?
  content_owner_data = content_item["links"]["content_owners"]
  ContentOwner.new(content_owner_data["title"], content_owner_data["base_path"])
else
  content_owner_data = content_item["details"]["content_owner"]
  ContentOwner.new(content_owner_data["title"], content_owner_data["href"])
end
```